parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -405,6 +405,7 @@ ensureKubelet() {
     if [ -n "${AZURE_ENVIRONMENT_FILEPATH}" ]; then
         echo "AZURE_ENVIRONMENT_FILEPATH=${AZURE_ENVIRONMENT_FILEPATH}" >> "${KUBELET_DEFAULT_FILE}"
     fi
+    chmod 0600 "${KUBELET_DEFAULT_FILE}"
```

Comment:
i suppose we do this for all files lower down this function but missed it for /etc/default/kubelet